### PR TITLE
add popover and create a test popover link out

### PR DESCRIPTION
**Pull request by akristen**
nerdgraph explorer popover

### CODE REVIEW

File: src/data/popovers_en.json [hunk 1]
```diff
@@ -139,5 +139,16 @@
       "primaryButton": "Read the tutorial",
       "primaryButtonUrl": "https://docs.newrelic.com/docs/tutorial-create-alerts/create-new-relic-alerts/"
     }
-  }
+  },
+  "nerdgraphExplorer": {
+    "inlineText": "NerdGraph",
+    "popover": {
+      "header": "NerdGraph is our GraphQL API that lets you make configurations, query your New Relic data, and access account information.",
```
Comment (austin-schaefer) [inline src/data/popovers_en.json:146]:
```suggestion
      "header": "NerdGraph is our GraphQL API to query your data, and configure your account and features.",
```

File: src/data/popovers_en.json [hunk 2]
```diff
@@ -139,5 +139,16 @@
...
+      "text": "NerdGraph is one of our main APIs for accessing New Relic data. You can access the endpoint by making calls in your code, or by using our NerdGraph API explorer.",
```
Comment (austin-schaefer) [inline src/data/popovers_en.json:147]:
```suggestion
      "text": "NerdGraph is our primary APIs for accessing New Relic data. You can access the endpoint by making calls in your code, or by using our NerdGraph API explorer.",
```

File: src/data/popovers_en.json [hunk 3]
```diff
@@ -139,5 +139,16 @@
       "primaryButton": "Read the tutorial",
       "primaryButtonUrl": "https://docs.newrelic.com/docs/tutorial-create-alerts/create-new-relic-alerts/"
     }
-  }
+  },
+  "nerdgraphExplorer": {
```
Comment (austin-schaefer) [inline src/data/popovers_en.json:143]: Let's code it as `nerdGraph` or similar, not explorer.

**Review by austin-schaefer — DISMISSED** [review]

File: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx [hunk 4]
```diff
@@ -180,7 +180,7 @@ To get started with API keys:
 
 You can view and manage most API keys from the [API keys UI page](https://one.newrelic.com/api-keys). To find that: from the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings/#user-menu), click <DoNotTranslate>**API keys**</DoNotTranslate>. You can also:
 
-* [Use NerdGraph explorer to view and create](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph#explorer) user keys
+* Use <InlinePopover type="nerdgraphExplorer" /> to view and create user keys
```
Comment (akristen) [inline src/content/docs/apis/intro-apis/new-relic-api-keys.mdx:183]:
```suggestion
* Use <InlinePopover type="nerdgraph" /> to view and create user keys
```

File: src/data/popovers_en.json (hunk 3 above)
Comment (akristen) [inline src/data/popovers_en.json:143]:
```suggestion
  "nerdgraph": {
```

File: src/data/popovers_en.json [hunk 5]
```diff
@@ -139,5 +139,16 @@
       "primaryButton": "Read the tutorial",
       "primaryButtonUrl": "https://docs.newrelic.com/docs/tutorial-create-alerts/create-new-relic-alerts/"
     }
-  }
+  },
+  "nerdgraph": {
+    "inlineText": "NerdGraph",
```
Comment (akristen) [inline src/data/popovers_en.json:144]:
```suggestion
    "inlineText": "NerdGraph API",
```

File: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx [hunk 6]
```diff
@@ -180,7 +180,7 @@ To get started with API keys:
 
 You can view and manage most API keys from the [API keys UI page](https://one.newrelic.com/api-keys). To find that: from the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings/#user-menu), click <DoNotTranslate>**API keys**</DoNotTranslate>. You can also:
 
-* [Use NerdGraph explorer to view and create](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph#explorer) user keys
+* Use <InlinePopover type="nerdgraph" /> to view and create user keys
```
Comment (rhetoric101) [inline src/content/docs/apis/intro-apis/new-relic-api-keys.mdx:183]: @akristen I know you didn't compose these two bullets, but aren't they saying the same thing?

I had recently written a revamp of this page that moved this "Manage" section to the list in the introduction about managing keys. 

Also, you could move that reference to Account ID into the table of keys. There's no reason to have it orphaned like this.

**Review by rhetoric101 — COMMENTED** [review]
@akristen I took a spin through this. Let's chat about these related changes you might make.

**Pull request closed without merging**